vault/resource_cert_auth_backend_role.go

```diff
@@ -0,0 +1,264 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+
+	"github.com/hashicorp/vault/api"
+)
+
+func certAuthBackendRoleResource() *schema.Resource {
+	return &schema.Resource{
+		SchemaVersion: 1,
+
+		Create: certAuthResourceWrite,
+		Update: certAuthResourceUpdate,
+		Read:   certAuthResourceRead,
+		Delete: certAuthResourceRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
```

Comment:
Would it be possible to strip the redundant field name declarations here and on the rest of the fields?

Reply:
@tyrannosaurus-becks - I'm not sure I follow what you mean re: redundant field names here.

Reply:
Sorry, I didn't phrase it very well. My code editor complains if, in a map like `map[string]*schema.Schema`, then `&schema.Schema` is also declared below. Here's a visual example:
![screenshot from 2018-09-14 13-54-19](https://user-images.githubusercontent.com/1152361/45574779-f9db4e00-b825-11e8-8a60-1bee59f22035.png)
It's underlining and greying out those words because they can be safely omitted.

I know it hasn't always been that way, but it would be great to go that way going forward. I'm thinking of doing a quick pass through the pre-existing code updating stuff like that.